### ISSUE TITLE
Attempting to place road node, doesnt place, selects terrain

### ISSUE THREAD

**Issue by JoshCodeStuff** [MicroGSD/RoadArchitect#72]
Unity 2019.2.10f1

So basically, I hold control and click to place a node, but it doesn't place, instead it selects the terrain behind it. I am on the linux version if that changes anything.

**Comment by embeddedt** [comment]
Do you have the road system object selected when you attempt to create a node? Did you restart Unity after installing Road Architect? Can you try [my fork](https://github.com/embeddedt/RoadArchitect) and see if it works?

**Comment by JoshCodeStuff** [comment]
> Do you have the road system object selected when you attempt to create a node? Did you restart Unity after installing Road Architect? Can you try [my fork](https://github.com/embeddedt/RoadArchitect) and see if it works?

Ill give it a shot, thanks man!

**Comment by JoshCodeStuff** [comment]
EDIT: this still happens, ot sure whats up, I am going to test an older version of unity

**Comment by embeddedt** [comment]
I have used it with 2019.2 on Linux before and it worked fine, so I'm surprised that you are having issues.

**Comment by JoshCodeStuff** [comment]
> I have used it with 2019.2 on Linux before and it worked fine, so I'm surprised that you are having issues.

Huh that's very weird. maybe it has to do with assets I have imported currently?

**Comment by FritzsHero** [comment]
Hello JoshCodeStuff,

I tested it today at 2019.2.4f1 and got some problems.
First of all, did you place the RoadArchitect files into the Assets/RoadArchitect folder?
This is the only **fully** supported path for the RoadArchitect files as of now.

For me my 2019.2.4f1 problem on windows was fixed after updating all installed packages in the package manager.
Maybe updating this packages helps?


Best Regards,

Fritzs

**Comment by JoshCodeStuff** [comment]
> Hello JoshCodeStuff,
> 
> I tested it today at 2019.2.4f1 and got some problems.
Yes to this > First of all, did you place the RoadArchitect files into the Assets/RoadArchitect folder?
> This is the only **fully** supported path for the RoadArchitect files as of now.
> 
> For me my 2019.2.4f1 problem on windows was fixed after updating all installed packages in the package manager.
> Maybe updating this packages helps?
> 
> Best Regards,
> 
> Fritzs

I deleted and reinstalled it and it works now. not sure how, thank you everyone!

**Comment by adrenak** [comment]
I had the same issue and decided to try to debug the code. Turns out the [OnSceneGUI method doesn't run at times](http://answers.unity.com/answers/1418260/view.html) and the handling of control+click happens there.

I was using multiple inspectors on multiple monitors. I just changed to the default editor layout and it started working. Then I changed back to my preferred editor layout and it kept working.
